fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/api/rdf/HttpResourceConverter.java

```diff
@@ -225,6 +227,8 @@ private String asString(final Resource resource, final Map<String, String> value
                 throw new InvalidResourceIdentifierException("Path contains empty element! " + path);
             }
             return path;
+        } else if (isRootWithoutTrailingSlash(resource)) {
```

Comment:
Yeah, I get the `if` part of `else if`. I am wondering about the `else` part. I prefer just a series of `if () return x` which makes the case structure a little more clear. Maybe that's less clear to others?